visidata/clipboard.py

```diff
@@ -74,6 +74,11 @@ def syscopyCells_async(sheet, cols, rows, filetype):
 
     # use NTF to generate filename and delete file on context exit
     with tempfile.NamedTemporaryFile(suffix='.'+filetype) as temp:
+        # Windows won't open a file which is already open.
+        # I'd prefer to reuse the open file handle like we do for syscopyValue
+        # above, but vd.SaveSheets expects to open a file from a file path, and
```

Comment:
visidata.Path can take an `fptext` argument which might/should work for this case.

Reply:
Oh fun! I'll see if I can get that working.

Reply:
I had to add a `write` method to `RepeatFile`, but it appears to work now with `fptext`. I've only tested it on Mac though.